### DISCUSSION TITLE
Meta MyoSuite 人工智慧平台，幫助研發義肢、復健及手術技術

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#667] [科技新知]
Facebook 母公司 Meta 23日發表全新的人工智慧平台 MyoSuite。Meta 表示，MyoSuite 除了能增加元宇宙使用者的體驗感，提升細膩及逼真程度外，更能協助研發義肢、復健和手術技術，可在醫療界廣泛使用。
MyoSuite 是一個透過機器學習的系統，能模仿人類神經線在運動時的移動情況，再模擬出逼真的肌肉骨骼，其模擬速度較其他同類型系統快 4,000 倍。
https://technews.tw/2022/05/31/meta-myosuite/

**Comment by sys511613** [comment]
有了這項技術後，對於之後的義肢、復健等都會很有幫助

**Comment by yucheng1028** [comment]
元宇宙還可以做真多事情

**Comment by alexthebigpp** [comment]
義肢、義眼我認為都是比較好製造出的人造器官的，期待以後可以做出人造內臟。

**Comment by yenwei12138** [comment]
沒想到元宇宙的發展不只影響虛擬世界，連現實的醫療也能幫助到

**Comment by 110409538** [comment]
科技的進步可以體現在醫學上.

**Comment by harrylee1971** [comment]
醫學的進步肯定是無窮止盡的，傷病只會有更多，但也促進著科技的進步去治療人類
